inputs/values/dockers.json

```diff
@@ -28,8 +29,8 @@
   "cnmops-virtual-env": "us.gcr.io/broad-dsde-methods/gatk-sv/cnmops-virtual-env:2022-06-10-v0.23-beta-9c6fbf56",
   "sv-pipeline-virtual-env": "us.gcr.io/broad-dsde-methods/gatk-sv/sv-pipeline-virtual-env:2022-06-10-v0.23-beta-9c6fbf56",
   "samtools-cloud-virtual-env": "us.gcr.io/broad-dsde-methods/gatk-sv/samtools-cloud-virtual-env:2022-06-10-v0.23-beta-9c6fbf56",
-  "sv-utils-env": "us.gcr.io/broad-dsde-methods/gatk-sv/sv-utils-env:2022-09-09-v0.25.1-beta-cd5ce7e0",
-  "sv_utils_docker": "us.gcr.io/broad-dsde-methods/gatk-sv/sv-utils:2022-09-21-v0.26.1-beta-c5e6abc8",
+  "sv-utils-env": "us.gcr.io/broad-dsde-methods/markw/sv-utils-env:mw-concordance-7251507",
+  "sv_utils_docker": "us.gcr.io/broad-dsde-methods/markw/sv-utils:mw-concordance-7251507",
   "gq_recalibrator_docker": "us.gcr.io/broad-dsde-methods/tbrookin/gatk:0a7e1d86f",
-  "str": "us.gcr.io/broad-dsde-methods/gatk-sv/str:2022-09-15-v0.25.1-beta-b53e58af"
```

Comment:
I suppose this PR is not related to EH, if so, I wonder how `build_docker.py` ended up incorrectly updating this, while it worked correctly when invoked by the bot.

Reply:
Had to do with the way I invoked build_docker.py - my master branch was out of date on the machine where I did the build but my working branch was rebased on the latest master, which contained your recent changes to some str code. I used "master" as the current commit - so it saw the str updates relative to the outdated master and rebuilt the docker.

Reply:
That explains it, thanks!